### PR TITLE
feat: add galoy auth chart

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: charts/galoy-auth/charts/auth-backend/values.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ingress:
+  enabled: false
+service:
+  port: 80
+  type: ClusterIP
+authEndpoint: api/login
+graphqlUri: http://localhost:4002/graphql
+graphqlSubscriptionUri: wss://localhost:4002/graphql
+supportEmail: support@galoy.io
+host: "0.0.0.0"
```
Comment (bodymindarts) [inline charts/galoy-auth/charts/auth-backend/values.yaml:14]: why is this exposed?

File: charts/galoy-auth/charts/auth-backend/values.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+image:
+  repository: us.gcr.io/galoy-org/web-wallet
+  digest: "sha256:ab4f308adba7241ea4ab71bc520b7e4fa404ba86fc65f77bb9d37d0ad19b63fc"
+  git_ref: "c9df056" # Not used by helm
+ingress:
+  enabled: false
+service:
+  port: 80
+  type: ClusterIP
+authEndpoint: api/login
```
Comment (bodymindarts) [inline charts/galoy-auth/charts/auth-backend/values.yaml:10]: I think this should be the whole url not just a path

**Review by bodymindarts — APPROVED** [review]

File: charts/galoy-auth/charts/auth-backend/values.yaml (hunk 1 above)
Comment (krtk6160) [reply charts/galoy-auth/charts/auth-backend/values.yaml]: removed

File: charts/galoy-auth/charts/auth-backend/values.yaml (hunk 2 above)
Comment (krtk6160) [reply charts/galoy-auth/charts/auth-backend/values.yaml:10]: Not sure if this is being used, or will be used in auth backend. Will fix in a later PR if it's used.

**Pull request merged**